app/notifications/validators.py

```diff
@@ -199,7 +199,7 @@ def check_email_annual_limit(service: Service, requested_emails=0):
     if not send_exceeds_annual_limit:
         # Will this send put the service right at their limit?
         if send_reaches_annual_limit and not annual_limit_client.check_has_over_limit_been_sent(service.id, SMS_TYPE):
```

Comment:
Should this line be `EMAIL_TYPE` ?
`annual_limit_client.check_has_over_limit_been_sent(service.id, SMS_TYPE):`

Reply:
Yes, yes it should thanks for catching this.